ldb/models.py

```diff
@@ -182,9 +238,11 @@ def age(self):
             return today.year - born.year
 
     def save(self):
+        self._membership_status = self.membership_status
```

Comment:
Gaan we dan de admin stukmaken omdat het net ietsjes mooier is om in clean te zetten?